apps/web/src/components/editor/table-of-contents.tsx

```diff
@@ -102,10 +104,10 @@ function TableOfContents(props: TableOfContentsProps) {
         display: "flex",
         position: "absolute",
         right: 0,
-        top: TITLE_BAR_HEIGHT,
+        top: 32,
```

Comment:
Why this change?

Reply:
`TITLE_BAR_HEIGHT` is 0 for web and since I moved the TOC a little below, I had to give some value to `top`. But now TOC isn't in absolute position but rather rendered to the right of editor. Hence, I've reverted this change as not needed anymore.